bin/install-wp-tests.sh

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env bash
```

Comment:
assuming this is just out of the box script from wordpress for setting up tests.